### ISSUE TITLE
nmstatectl show ethtool flags randomly sorted

### ISSUE THREAD

**Issue by eikef** [nmstate/nmstate#2343]
## Describe the bug
`nmstatectl show` shows `ethtool` features unsorted. Unfortunately, the sorting changes on every invocation; two chained executions of the `nmstatectl show` differ wildly in output (though not in actual semantics)

## Nmstate version
- Version 2.2.10

## NetworkManager version
- Version 1.42.2

## Platform
- Platform: EL9

## How reproducible
- 100%

## Steps to Reproduce
```
nmstatectl show
nmstatectl show
```

or 

```
nmstatectl show > a
nmstatectl show > b
diff -u a b
```

## Nmstate output
- Expected output:

Same ordering of options

- Current output:
```
(...)
  ethtool:
    feature:
      rx-gro-hw: true
      rx-udp-gro-forwarding: false
      tx-checksum-ip-generic: true
      tx-tcp-segmentation: true
      tx-tcp6-segmentation: true
      rx-gro: true
      tx-generic-segmentation: true
      rx-gro-list: false
      tx-nocache-copy: false
      tx-tcp-mangleid-segmentation: false
      tx-tcp-ecn-segmentation: true
(...)
  ethtool:
    feature:
      rx-gro: true
      tx-tcp-ecn-segmentation: true
      tx-checksum-ip-generic: true
      tx-tcp-mangleid-segmentation: false
      tx-tcp6-segmentation: true
      tx-tcp-segmentation: true
      tx-nocache-copy: false
      rx-gro-hw: true
      rx-udp-gro-forwarding: false
      rx-gro-list: false
      tx-generic-segmentation: true
```
or 

```
--- a	2023-05-15 12:54:00.145593896 +0000
+++ b	2023-05-15 12:54:01.178593896 +0000
@@ -50,17 +50,17 @@
     enabled: false
   ethtool:
     feature:
-      rx-gro: true
       tx-tcp-mangleid-segmentation: false
       tx-generic-segmentation: true
-      tx-tcp-segmentation: true
-      rx-gro-hw: true
-      rx-gro-list: false
-      tx-nocache-copy: false
-      tx-checksum-ip-generic: true
-      tx-tcp6-segmentation: true
+      rx-gro: true
       tx-tcp-ecn-segmentation: true
+      tx-tcp6-segmentation: true
+      tx-nocache-copy: false
+      rx-gro-list: false
       rx-udp-gro-forwarding: false
+      tx-checksum-ip-generic: true
+      tx-tcp-segmentation: true
+      rx-gro-hw: true
     coalesce:
       rx-frames: 1
       tx-frames: 1
```

### Additional context

This is a minor annoyance, but it does make diffing (possibly historic) states harder.

**Comment by ffmancera** [comment]
+1

**Comment by cathay4t** [comment]
Considering this is a not a simple task, so I have fixed this in PR #2389 .
